affine/__init__.py

```diff
@@ -255,42 +245,39 @@ def rotation(cls, angle: float, pivot=None):
             return tuple.__new__(cls, (ca, -sa, 0.0, sa, ca, 0.0, 0.0, 0.0, 1.0))
         else:
             px, py = pivot
+            c = px - px * ca + py * sa
+            f = py - px * sa - py * ca
```

Comment:
The code without the extra assignment should be faster, yes? And it's not that hard to read.